app/controllers/v1/applications_controller.rb

```diff
@@ -14,4 +20,9 @@ def show
     options = { include: [:applicant] }
     render json: LegalAidApplicationSerializer.new(application, options).serialized_json
   end
+
+  def process_proceeding_types(_application)
```

Comment:
If you address the [previous comment](https://github.com/ministryofjustice/laa-apply-for-legalaid-api/pull/18/files#r214990391), this might not be necessary at all, but if you're not using `application` in the method it shouldn't be passed as an argument at all.

Reply:
done

Reply:
done